spec/src/main/asciidoc/changelog.adoc

```diff
@@ -63,10 +70,70 @@
 ** Removed `getSimpleTimers()` method
 ** Removed `getSimpleTimers(MetricFilter filter)` method
 
+* Updated DefaultMetadata class
+** Removed displayName from constructor
+** Removed `getDisplayname()` method
+** Removed `displayName()` method
+
+* Updated Metadata class
+** Removed `getDisplayname()` method
+** Removed `displayName()` method
+
+* Updated MetadataBuilder class
+** Removed `withDisplayName(String displayName)` method
+
+* Updated Snapshot class
+** Removed `getValue(double quantile)` method
+** Removed `getValues()` method
+** Removed `get75thPercentile()` method
+** Removed `get95thPercentile()` method
+** Removed `get98thPercentile()` method
+** Removed `get999thPercentile()` method
+** Removed `get99thPercentile()` method
+** Removed `getMedian()` method
+** Removed `getMin()` method
+** Removed `getStdDev()` method
+** Modified `size()` method to return long
+** Modified `getMax()` method to return double
+
+* Updated Gauge class
+** can now only work with types that extend Number
+
 * Updated MetricType class
-** Removed `CONCURRENT_GAUGE("concurrent gauge", ConcurrentGauge.class)` enum
-** Removed `METERED("meter", Meter.class)` enum
-** Removed `SIMPLE_TIMER("simple timer", SimpleTimer.class)` enum
+** Removed `CONCURRENT_GAUGE` enum
+** Removed `METERED` enum
+** Removed `SIMPLE_TIMER` enum
+
+=== API/SPI Changes
+* Updated Snapshot class
+** Added `percentileValues()` method
```

Comment:
Should we also include that we added the new `Snapshot.PercentileValue` innerclass

Reply:
yes, good idea.